utils/update_build_version.py

```diff
@@ -62,7 +62,8 @@ def describe(dir):
         try:
             return command_output(['git', 'rev-parse', 'HEAD'], dir).rstrip()
         except:
-            return 'unknown hash, ' + datetime.date.today().isoformat()
+            return 'unknown hash, {}'.format(
+                datetime.date.today().isoformat()).encode('ascii')
```

Comment:
I think it would be much more useful to leave this as is and add `decode` to lines 60 and 63.

But if you want this, then please change the blurb on line 54.